### ISSUE TITLE
need to add alt tag to image 

### ISSUE THREAD

**Issue by Cibiyanna26** [team-devoice/Effitrack#14]
An <img> element has no alt attribute.
An <input type="image"> element has no alt, aria-label or aria-labelledby attribute or they hold an empty string.
An <area> element within an image map has no alt, aria-label or aria-labelledby attribute.
An <object> element has no inner text, title, aria-label or aria-labelledby attribute.

**Comment by Cibiyanna26** [comment]
so whenever you guys are free free to add alt tag to the unnoticed images
